### ISSUE TITLE
Serve React App from subdirectory in the project!

### ISSUE THREAD

**Issue by KGmajor** [YearOne-Prep/YearOne-prep-challenges#46]
I'd like to use the information from the topics yaml to serve on a static React site. In order to access the yaml, the React application should live in a subdirectory and access the same yaml as the posts that go to the community.

Describe the solution you'd like
Use create react app, but keep the react files serving from a subdirectory named react.

Additional context
We'll probably use a tool like render, vercel, or heroku to deploy the static page.

**Comment by mairagee524** [comment]
I may be able to take this on! From my understanding, I just need to create-react-app, right?

**Comment by KGmajor** [comment]
Yep @mairagee524! And make sure it's accessible and able to render from the subdirectory!
